CHANGELOG.md

```diff
@@ -7,14 +7,25 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+- Add feature to cast from YouTube mobile app (on phone, tablet)
 
 ### Changed
+- Hide Pocket video tile on non-EN locales
+    - Pocket does not yet provide content for other languages
+
 - Sentry crash reports include a UUID to distinguish users so we can determine if it's 1 user crashing 100 times or 100 users crashing 1 time each. This identifier is only used for Sentry and can not be correlated with telemetry interaction data. See [fire TV Sentry docs](https://github.com/mozilla-mobile/firefox-tv/wiki/Crash-reporting-with-Sentry) for more details. (#817)
 
 ### Fixed
 - Blank screen when pressing back from a full-screened video
 - Infrequent crash caused by initialization logic (#1159)
 
+## [3.0.2] - 2018-10-30
+Version-bump only, released adding *Stick Gen 1 & 2* in addition to *Fire TV (Gen 1, 2, 3), Cube, Element 4k (pendant)*
+
+## [3.0.1] - RC candidate, did not release
+### Fixed
+Issue that would cause browsing history to be lost when navigating to device home screen
```

Comment:
nit: I try to add issue numbers to fixed items, e.g. - Issue that causes regressions (#450000)